### PR TITLE
Adds FAQ about timestamps

### PR DESCRIPTION
**Pull request by deandre**

### CODE REVIEW

File: src/content/troubleshooting/faq/timestamps.mdx [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+layout: "../../../layouts/FAQLayout.astro"
+sidebar: { hide: true }
+title: How are timestamps displayed in Chromatic?
+section: usage
+---
+
+# How are timestamps displayed in Chromatic?
+
+While our capture and webapp servers are set to UTC, all timestamps in Chromatic are displayed in your system's local time.
```
Comment (winkerVSbecks) [inline src/content/troubleshooting/faq/timestamps.mdx:10]: To clarify, Chromatic's web app will show local time in Canvas view. But snapshots will show server time, right? If so, might be worth calling that out.

File: src/content/troubleshooting/faq/timestamps.mdx [hunk 2]
```diff
@@ -0,0 +1,10 @@
+---
+layout: "../../../layouts/FAQLayout.astro"
+sidebar: { hide: true }
+title: How are timestamps displayed in Chromatic?
+section: usage
+---
+
+# How are timestamps displayed in Chromatic?
+
+While our server infrastructure is in UTC, Chromatic displays all timestamps in your system's local time.
```
Comment (winkerVSbecks) [inline src/content/troubleshooting/faq/timestamps.mdx:10]:
```suggestion
While our server infrastructure is in UTC, Chromatic web app displays all timestamps in your system's local time.
```

File: src/content/troubleshooting/faq/timestamps.mdx [hunk 3]
```diff
@@ -0,0 +1,10 @@
+---
+layout: "../../../layouts/FAQLayout.astro"
+sidebar: { hide: true }
+title: How are timestamps displayed in Chromatic?
+section: usage
+---
+
+# How are timestamps displayed in Chromatic?
```
Comment (winkerVSbecks) [inline src/content/troubleshooting/faq/timestamps.mdx:8]:
```suggestion
# How are timestamps displayed in Chromatic web app?
```

**Review by winkerVSbecks — APPROVED** [review]

**Comment by winkerVSbecks** [comment]
@deandre thanks!

File: src/content/troubleshooting/faq/timestamps.mdx (hunk 1 above)
Comment (deandre) [reply src/content/troubleshooting/faq/timestamps.mdx]: good point — this concerns timestamps in the web app (e.g., build and billing screens), not so much canvas/snapshot.

Comment (winkerVSbecks) [reply src/content/troubleshooting/faq/timestamps.mdx]: Okay that makes a lot of sense. I was confused for some reason. My first thought was timestamps in test code 🤦🏽‍♂️ 

Maybe add the "(e.g., build and billing screens)" to the text?

Comment (deandre) [reply src/content/troubleshooting/faq/timestamps.mdx]: for sure, thanks varun!

**Pull request merged**